apps/rule-manager/client/src/ts/components/RulesTable.tsx

```diff
@@ -59,10 +59,6 @@ const createColumns = (editRule: (ruleId: number) => void): Array<EuiBasicTableC
       field: 'ruleType',
       name: 'Type',
     },
-    {
-      field: 'externalId',
-      name: 'ID'
-    },
```

Comment:
Out of interest, why are we removing this column?

Reply:
Ooh, should've mentioned this in the description – it makes room for the 'status' column in our table, and also fits our user story `✅I want to not have to worry about secondary rule information (ie. ID)'` (story doc [here.](https://docs.google.com/document/d/1dPSplzFR8RkQyPP0TSXlGkahmFU1q-znKneflursWVs/edit)) Description updated – hope that makes sense!

Reply:
Cool, makes sense! 👌